emeis/core/models.py

```diff
@@ -118,3 +145,54 @@ def is_authenticated(self):
         This is a way to tell if the user has been authenticated in templates.
         """
         return True
+
+
+class Scope(UUIDModel):
+    name = LocalizedCharField(_("scope name"), blank=False, null=False, required=False)
+    description = LocalizedTextField(
+        _("scope description"), null=True, blank=True, required=False
+    )
+    parent = models.ForeignKey(
```

Comment:
I suggest using `django-mptt` for hierarchical models. This will speed up querying children of a scope and a few other common access patterns by implementing the "Nested Set" pattern.

https://github.com/django-mptt/django-mptt

Reply:
Why didn't I think of that!?

Done.